src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -176,7 +176,7 @@ private static TemplateEnd findTemplateEnd( final GATKRead read ) {
         public SplitRead( final GATKRead read, final ReadMetadata metadata, final boolean atStart ) {
             super(read, metadata, atStart ? read.getStart() : read.getEnd(), UNCERTAINTY);
             cigar = read.getCigar().toString();
-            if ( cigar == null ) throw new GATKException("Read has no cigar string.");
+            if ( cigar == null ) throw new GATKException("Read has no forwardStrandCigar string.");
```

Comment:
I believe this change is an IDE auto-rename error, as this was a different part of the SV pipeline...